DailySelfie/app/src/main/java/com/teamagam/dailyselfie/MainActivity.java

```diff
@@ -19,4 +36,67 @@ public boolean onCreateOptionsMenu(Menu menu) {
         inflater.inflate(R.menu.options_menu, menu);
         return true;
     }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.action_take_picture:
+                dispatchTakePictureIntent();
+                return true;
+
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    private void dispatchTakePictureIntent() {
+        Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+        // Ensure that there's a camera activity to handle the intent
+        if (takePictureIntent.resolveActivity(getPackageManager()) != null) {
+            // Create the File where the photo should go
+            File photoFile = null;
+            try {
+                photoFile = createImageFile();
+            } catch (IOException e) {
+                Toast.makeText(this, e.getMessage(), Toast.LENGTH_SHORT).show();
+            }
+            // Continue only if the File was successfully created
+            if (photoFile != null) {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
why do address this SDK?
have you checked the app in this SDK?

you need to be focused on min SDK and max SDK (.N is way over the scope of this project),